falcon/__init__.py

```diff
@@ -529,6 +530,7 @@
 from falcon.status_codes import HTTP_BAD_GATEWAY
 from falcon.status_codes import HTTP_BAD_REQUEST
 from falcon.status_codes import HTTP_CONFLICT
+from falcon.status_codes import HTTP_CONTENT_TOO_LARGE
```

Comment:
not in all

Reply:
Doh, that's not good. I'll address it in the alpha1 PR.